### PR TITLE
Fix multiple SelectorFormat issues in views/_signup.scss

### PR DESCRIPTION
**Pull request by rferromoreno**
Fixing issue #80 but also other "SelectorFormat" warnings according to [this link](https://codeclimate.com/github/chatwoot/chatwoot/app/javascript/src/assets/scss/views/_signup.scss#issue_5d87481187cf1900010005ca).

Only for views/_signup.scss

Other files like _login.scss seems to have the same warnings.

### CODE REVIEW

**Comment by pranavrajs** [comment]
@rferromoreno Thanks for opening the PR. Just opening a discussion with @nithindavid here.

@nithindavid This seems like CodeClimate has SelectorFormat rules associated with it. Could you please take a look and see if any further action is required in this PR?

File: app/javascript/src/assets/scss/views/_signup.scss [hunk 1]
```diff
@@ -1,25 +1,25 @@
 .signup {
   // margin-top: $space-larger*1.2;
 
-  .signup__hero {
+  .signup-hero {
```
Comment (pranavrajs) [inline app/javascript/src/assets/scss/views/_signup.scss]: We follow BEM, so `.signup--hero` should work here.

Comment (rferromoreno) [reply app/javascript/src/assets/scss/views/_signup.scss]: Okay. I was reviewing other files and saw that some places are used with a single hyphen. Was trying get a convention there.

Should I also apply this change to classes like `signup-box`, `accept-terms` aswell?

Comment (pranavrajs) [reply app/javascript/src/assets/scss/views/_signup.scss]: Yes, please

Comment (rferromoreno) [reply app/javascript/src/assets/scss/views/_signup.scss]: Okay, done!

**Comment by pranavrajs** [comment]
@rferromoreno Could you please share your details here? We would like to send you a sticker pack for creating the PR to our repo. We will start shipping it by the end of October.

**Comment by rferromoreno** [comment]
@pranavrajs Wow! I wasn't expecting that. It's a very nice detail!
Thank you very much!

**Pull request merged**